Jenkinsfile

```diff
@@ -10,39 +10,102 @@ pipeline {
     buildDiscarder(logRotator(numToKeepStr: '5', artifactNumToKeepStr: '5'))
     timestamps()
   }
+  parameters {
+    booleanParam(name: 'Build', defaultValue: true, description: 'Includes Build app and Tests')
+    booleanParam(name: 'Release', defaultValue: false, description: 'Includes Tagging and Delivery')
+    booleanParam(name: 'Deployment', defaultValue: false, description: 'Deploy a new version of App')
+    booleanParam(name: 'SetNewTag', defaultValue: false, description: 'Auto-increasing version')
+    choice(name: 'AppVersion', choices: ['Minor', 'Middle', 'Major'], description: 'Pick Version Tag')
+    choice(name: 'NewVersionTrafficWeight', choices: [0, 1, 2, 3, 4, 5, 6, 7, 8, 9], description: 'Amount of traffic to the new vesion of the App')
+  }
   environment {
     ECRURI = '054017840000.dkr.ecr.us-east-1.amazonaws.com'
+    AWSRegion = 'us-east-1'
     AppRepoName = 'snakes'
-    OPSRepoURL = 'https://github.com/IYermakov/DevOpsA3Training.git'
-    OPSRepoBranch = 'ecs-snakes'
-    Tag = ""
-    Email = 'vecinomio@gmail.com'
+    OPSRepoURL = 'git@github.com:IYermakov/DevOpsA3Training.git'
+    OPSRepoBranch = 'weighted-tgs'
+    BuildAndTest = "${params.Build}"
+    Release = "${params.Release}"
+    Deployment = "${params.Deployment}"
+    Tag = '1.0.0'
```

Comment:
extract to parameter with default value

Reply:
In my opinion, User shouldn't has an opportunity to change the value of this variable, because this is a reserve case: if there is no any releases in the repository.

Reply:
ok, so please rename it

Reply:
Changed the name of the variable